libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -39,10 +39,6 @@ describe('AccordionItemComponent', () => {
       expect(spectator.query('.title')).toHaveExactTrimmedText(expectedText);
     });
 
-    it('should have correct tabindex', () => {
```

Comment:
Would be good to add a couple of tests, especially for the accordion -> accordion-item integration, that validates that setting the property results in the correct attributes on the heading element.